tests/src/Functional/InstalledConfigurationTest.php

```diff
@@ -0,0 +1,436 @@
+<?php
+
+namespace Drupal\Tests\thunder\Functional;
+
+use Drupal\Component\Utility\NestedArray;
+use Drupal\Core\Config\Schema\SchemaCheckTrait;
+use Drupal\Tests\BrowserTestBase;
+use Drupal\thunder\ThunderTestTrait;
+
+/**
+ * Test for checking of configuration after install of thunder profile.
+ *
+ * @package Drupal\Tests\thunder\Kernel
+ *
+ * @group Thunder
+ */
+class InstalledConfigurationTest extends BrowserTestBase {
```

Comment:
You can extend from ThunderBaseTest. Then you can remove ThunderTestTrait and the $profile variable